giskard_vision/landmark_detection/detectors/surrogate_functions.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from scipy.spatial import ConvexHull
+
+from giskard_vision.landmark_detection.tests.performance import Es, NMEMean
+from giskard_vision.landmark_detection.types import PredictionResult
+
+
+@staticmethod
```

Comment:
this decorator is only needed for class methods

Reply:
Actually as I use them as class attributes, they can't be called with self if they're not static